kmath-stat/src/commonMain/kotlin/space/kscience/kmath/series/DynamicTimeWarping.kt

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2018-2023 KMath contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package space.kscience.kmath.series
+
+import space.kscience.kmath.nd.*
+import space.kscience.kmath.nd.DoubleBufferND
+import space.kscience.kmath.nd.ShapeND
+import space.kscience.kmath.nd.ndAlgebra
+import space.kscience.kmath.operations.DoubleField
+import space.kscience.kmath.structures.DoubleBuffer
+import kotlin.math.abs
+
+public const val LEFT_OFFSET : Int = -1
+public const val BOTTOM_OFFSET : Int = 1
+public const val DIAGONAL_OFFSET : Int = 0
```

Comment:
What are these constants for? It should be documented and maybe marked internal.

Reply:
This constants for avoidong magical numbers there is no more sense in them.

Reply:
Added comments.